frontend/src/components/SchemaEditor/DatabaseTree.vue

```diff
@@ -493,13 +497,28 @@ const renderLabel = ({ option: treeNode }: { option: TreeNode }) => {
 
 // Render a 'menu' icon in the right of the node
 const renderSuffix = ({ option: treeNode }: { option: TreeNode }) => {
+  const instanceEngine = instanceStore.getInstanceById(
+    treeNode.instanceId
+  ).engine;
+
   if (treeNode.type === "database") {
-    return h(EllipsisIcon, {
-      class: "w-4 h-auto text-gray-600",
-      onClick: (e) => {
-        handleShowDropdown(e, treeNode);
-      },
-    });
+    if (instanceEngine === "MYSQL") {
+      return h(EllipsisIcon, {
+        class: "w-4 h-auto text-gray-600",
+        onClick: (e) => {
+          handleShowDropdown(e, treeNode);
+        },
+      });
+    }
+  } else if (treeNode.type === "schema") {
+    if (instanceEngine === "POSTGRES") {
+      return h(EllipsisIcon, {
+        class: "w-4 h-auto text-gray-600",
+        onClick: (e) => {
+          handleShowDropdown(e, treeNode);
+        },
+      });
+    }
```

Comment:
Of course not in this PR, but I strongly recommend us to extract this render function to a dedicated component. Ref: https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/frontend/src/views/sql-editor/TabList/TabItem/Suffix.vue